README.md

```diff
@@ -129,13 +107,13 @@ Feel free to check the [issues page](https://github.com/binodbhusal/Hello-Microv
 
 ## ⭐️ Show your support <a name="support"></a>
 
-If you like this project  please feel free to contact me for any kind of support.
+If you like this project please feel free to contact me for any kind of support.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-Original design idea by Cindy Shin in Behance.(https://www.behance.net/adagio07)
+We would like to thank our mentos and Microverse for their support in building the project.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Your Licence link should not lead to this 👇🏾 page. Instead, it should lead to a LICENSE.md file that contains your license statement. 

![image](https://user-images.githubusercontent.com/84156714/234565288-03b6a69f-2b70-43fa-8dcc-5a4c62270175.png)


### **LICENSE.md - License statement** 

```
 MIT License

Copyright (c) [year] [fullname] #Edit this with your name and partner's name

Permission is hereby granted, free of charge, to any person obtaining a copy
  of this software and associated documentation files (the "Software"), to deal
  in the Software without restriction, including without limitation the rights
  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
  copies of the Software, and to permit persons to whom the Software is
  furnished to do so, subject to the following conditions:
  
  The above copyright notice and this permission notice shall be included in all
  copies or substantial portions of the Software.
  
  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  SOFTWARE.

```